tutorials/solution_strategies.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<p>The first sentence is aspirational at the moment. Should we add a comment that, for now, this includes only a line search method? Or that the tutorial is under construction?</p><p> </p><p>Can we also add (somewhere) a short description of how to use this as a 'standard' global line search?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>